extensions.json

```
@@ -818,23 +818,22 @@
     {
       "id": "ms-vscode.js-debug",
       "repository": "https://github.com/microsoft/vscode-js-debug",
-      "version": "1.50.0",
-      "checkout": "v1.50.0",
-      "prepublish": "npm run compile && npm run package",
-      "extensionFile": "dist/js-debug.vsix"
+      "version": "1.50.1",
+      "checkout": "master",
+      "prepublish": "npm run compile && npm run package"
```

Oops, the upgrade script stripped `"extensionFile": "dist/js-debug.vsix"` from the extension parameters.

That's because this parameter is quite new, and we forgot to make `upgrade-extensions.js` explicitly keep it.

Could you please make `upgrade-extensions.js` keep the `extensionFile` parameter by:
1. Adding a `--extensionFile` CLI flag in [add-extension.js](https://github.com/open-vsx/publish-extensions/blob/master/add-extension.js) (e.g. everywhere you see the `prepublish` CLI option being used)
2. Adding the `--extensionFile` CLI flag in `upgrade-extensions.js` when the parameter exists:

https://github.com/open-vsx/publish-extensions/blob/6dcbd4864d9b8bf9b0c6384be0f79e1f77cf976d/upgrade-extensions.js#L50-L59

Also, please manually add `extensionFile` back here before we forget about it 😅:

```suggestion
      "prepublish": "npm run compile && npm run package",
      "extensionFile": "dist/js-debug.vsix"
```

---

@jankeromnes Please take a look at the changes I did in order to make sure I introduced the correct CLI tag in an expected way!

---

@alexventuraio Awesome! Your changes look good to me. 👍

Many thanks for taking the time to contribute this. 💯

---

All a pleasure to help something I like and find it useful for me! 😃